Tella/Components/ReloadButton.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ReloadButton.swift
+//  Tella
+//
+//  Created by gus valbuena on 1/31/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct ReloadButton: View {
```

Comment:
I think it's better to create a ReloadButton with ToolbarContent type instead to create another reloadButton function in View extension 


   ```
struct ReloadButtonn: ToolbarContent {
    var action: () -> Void
    var body: some ToolbarContent {
        ToolbarItem(placement: .navigationBarTrailing) {
            Button(action: action) {
                Image("arrow.clockwise")
                    .resizable()
                    .frame(width: 24, height: 24)
            }
        }
    }
 }
```